tests/unit_tests/test_median.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+import pytest
+
+from tests.utils import load_process_code, run_process
+
+
+@pytest.fixture
+def median_process_code():
+    return load_process_code("median")
+
+
+@pytest.mark.parametrize(
+    "example_input,expected_output",
+    [
+        ({"data": [1, 3, 3, 6, 7, 8, 9]}, 6),
+        ({"data": [1, 2, 3, 4, 5, 6, 8, 9]}, 4.5),
+        ({"data": [-1, -0.5, None, 1]}, -0.5),
+        ({"data": [-1, 0, None, 1], "ignore_nodata": False}, None),
+        ({"data": []}, None),
+        ({"data": [], "ignore_nodata": False}, None),
+        ({"data": [None, None, None, None]}, None),
+        ({"data": [None, None, None, None], "ignore_nodata": False}, None),
```

Comment:
add following tests
```
        ({"data": [1, 2, 3]}, 2),
        ({"data": [1, 3, 2]}, 2),
        ({"data": [1, 2, 3, 4]}, 2.5),
        ({"data": [2, 1, 3, 4]}, 2.5),
```
I think we can't make an assumption that input array is sorted

Reply:
Should be ok now, added sorting of array before calculating the median.

Added these test cases and 2 more of not sorted arrays.